README.md

```diff
@@ -54,15 +54,15 @@ docker build . --tag=nexus-web
 
 - `BASE_PATH`: The base of the app: i.e. `/staging/web` if hosted on `https://bbp-nexus.epfl.ch/staging/web` (default is `/`)
 - `HOST_NAME`: name of host where application is available from: i.e. `https://bbp-nexus.epfl.ch` (default is protocol + host where server is running from)
-- `CLIENT_ID`: The application name used for _OpenID Connect_ authentication
+- `CLIENT_ID`: The application name used for _OpenID Connect_ authentication (default is `nexus-web`)
 - `API_ENDPOINT`: The URL pointing to Nexus API. Default is '/'
-- `AUTH_ENDPOINT`: _OpenID Connect_ Authentication endpoint. Empty by default (no auth).
-- `LOGOUT_ENDPOINT`: _OpenID Connect_ Logout endpoint. Empty by default (no logout).
+- `SECURE`: Is nexus web running in https or not. Default is `true`
```

Comment:
SECURE should not be assumed to be true if we don't know the context, it should always be based on the URL scheme currently in use (detection on the server side) and fall back to false

Reply:
It cannot be based on the URL as, in our case for instance, the service is running in OpenShift, behind a reverse proxy. This flag is to is to fix that very issue we had in our dev environment for example.